### DISCUSSION TITLE
GPU sampler

### DISCUSSION THREAD

**Discussion by stelmo** [LCSB-BioCore/COBREXA.jl#528] [Ideas]
Will making a GPU based hit and run sampler speed things up? Investigate and add if so.

**Comment by exaexa** [comment]
Yes, sure, the sampling is in fact quite GPU-izable.

Might need a specialized sampling that benefits from the throughput though. (Would the ACHR thing that we have work better if we run it "much longer than now"?)

**Comment by stelmo** [comment]
Good question, I am not sure... It has been a while since I really delved into samplers etc. This seems to be a good review: https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0235393
